### PR TITLE
add support to configure number alignment and test project

### PR DESCRIPTION
**Pull request by Alexandre-Machado**
image

image

### CODE REVIEW

File: src/ConsoleTables/ConsoleTable.cs [hunk 1]
```diff
@@ -219,20 +254,30 @@ public void Write(Format format = ConsoleTables.Format.Default)
...
 
     public class ConsoleTableOptions
     {
         public IEnumerable<string> Columns { get; set; } = new List<string>();
         public bool EnableCount { get; set; } = true;
+
+        /// <summary>
+        /// Enable only from a list of objects
+        /// </summary>
+        public bool NumberRigthAligned { get; set; }
```
Comment (khalidabuhakmeh) [inline src/ConsoleTables/ConsoleTable.cs]: Would this be better as an enum value? `Left`, `Right`, `Center` for example.

Comment (Alexandre-Machado) [reply src/ConsoleTables/ConsoleTable.cs]: I thought about it, but string.format does not have center alignment

File: ConsoleTables.Tests/ConsoleTableTest.cs [hunk 2]
```diff
@@ -30,7 +30,7 @@ public void ShouldBeAvoidErrorOnToStringFromAddRows()
             var table = new ConsoleTable("one", "two", "three")
                 .AddRow(1, 2, 3)
                 .AddRow("this line should be longer", "yes it is", "oh")
-                .Configure(o => o.NumberRigthAligned = true)
+                .Configure(o => o.NumberAlignment = Alignment.Rigth)
```
Comment (khalidabuhakmeh) [inline ConsoleTables.Tests/ConsoleTableTest.cs]: You spelled `Right` wrong

File: src/ConsoleTables/ConsoleTable.cs [hunk 3]
```diff
@@ -242,4 +291,10 @@ public enum Format
         Alternative = 2,
         Minimal = 3
     }
+
+    public enum Alignment
+    {
+        Left,
+        Rigth
```
Comment (khalidabuhakmeh) [inline src/ConsoleTables/ConsoleTable.cs]: Misspelled `Right`

Comment (Alexandre-Machado) [reply src/ConsoleTables/ConsoleTable.cs]: thanks

**Comment by khalidabuhakmeh** [comment]
I added CI, you can rebase to make it pass, otherwise I can just merge and yolo the build in master

**Comment by khalidabuhakmeh** [comment]
Actually based on the details of the build your assertions are failing due to a difference between `\n` and `\r\n`. Argh!

File: ConsoleTables.Tests/ConsoleTableTest.cs (hunk 2 above)
Comment (Alexandre-Machado) [reply ConsoleTables.Tests/ConsoleTableTest.cs]: thanks

**Comment by khalidabuhakmeh** [comment]
Thank you so much, this should be released under the 2.3.0 version as soon as NuGet is done scanning it.

**Pull request merged**